test/katex-spec.js

```diff
@@ -1382,6 +1382,15 @@ describe("An HTML font tree-builder", function() {
         span = "<span class=\"mord mathbb\" style=\"color:blue;\">R</span>";
         expect(markup).toContain(span);
     });
+
+    it("should throw TypeError when the expression is of the wrong type", function() {
+        expect(function() {
+            katex.renderToString({badInputType: "yes"});
+        }).toThrowError(TypeError);
+        expect(function() {
+            katex.renderToString(["foo", {badInputType: "yes"}]);
+        }).toThrowError(TypeError);
+    });
```

Comment:
Just put a single test to [the parser test group](https://github.com/Khan/KaTeX/blob/a16ae7a5ebc97cfb610a2143254393918fd0650d/test/katex-spec.js#L199-L209). Something simple like `expect(new String("xy")).toParseLike("xy")`.